src/uct/cuda/cuda_ipc/cuda_ipc_ep.h

```diff
@@ -33,8 +33,8 @@ static inline khint_t uct_cuda_ipc_memh_hash_func(CUipcMemHandle seg)
     return hash_val;
 }
 
-#define uct_cuda_ipc_memh_hash_equal(_sg1, _sg2)  \
-    strncmp((const char *) &(_sg1), (const char *) &(_sg2), sizeof(CUipcMemHandle))
+#define uct_cuda_ipc_memh_hash_equal(_sg1, _sg2)                        \
```

Comment:
why need these spaces?

Reply:
This was because of emacs region indentation. These change was automatically applied for the macro.

Reply:
@brminich @yosefe 
Do I need to remove the spaces even if was automatically done by emacs?